homeassistant/config_entries.py

```diff
@@ -2447,6 +2447,7 @@ def _abort_if_unique_id_mismatch(
         self,
         *,
         reason: str = "unique_id_mismatch",
+        description_placeholders: Mapping[str, str] | None = None,
```

Comment:
Small change here to the signature of `_abort_if_unique_id_mismatch` to add a new keyword argument